string a_operation_pausing_reason
---
int64 a_operation_status
